### PR TITLE
Add validation

### PR DESCRIPTION
**Pull request by esikachev**
Add validetion and fix structure for this change

### CODE REVIEW

**Comment by esikachev** [comment]
ci +1

**Comment by ivanovmi** [comment]
Why you remove coverage section?

**Comment by esikachev** [comment]
@ivanovmi because we migrate to https://coveralls.io/

**Comment by esikachev** [comment]
update README. Tests will be pass, because i can't touch program files

File: shellper/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__author__ = 'evgeny'
```
Comment (kozlovpn) [inline shellper/tests/__init__.py]: I think this need to delete (esikachev)

**Comment by esikachev** [comment]
@kozlovpn you're right

**Comment by kozlovpn** [comment]
I think it`s +1

**Comment by esikachev** [comment]
+2 ( @ivanovmi )

**Pull request merged**